pkg/utils/fileHandling.go

```diff
@@ -0,0 +1,82 @@
+package utils
+
+import (
+	"fmt"
+	"io"
+	"io/ioutil"
+	"net/http"
+	"os"
+	"strings"
+
+	"github.com/pkg/errors"
+)
+
+// EditFile will edit the content of a file
+func EditFile(filepath, old, new string) error {
+	fileData, err := ioutil.ReadFile(filepath)
+	if err != nil {
+		return errors.Wrapf(err, "Failed to read the given file, due to:%v", err)
+	}
+	lines := strings.Split(string(fileData), "\n")
```

Comment:
Is this line split required, Can you check this https://gist.github.com/tdegrunt/045f6b3377f3f7ffa408

Reply:
split helps in counting the number of lines. 
It is helping in a case where we want to change the next line after we got a match.

Reply:
check line 46